unison-src/transcripts/link.output.md

```diff
@@ -196,27 +196,15 @@ myLibrary.h x = x + 3
   Note: The most recent namespace hash is immediately below this
         message.
   
-  ⊙ #jlbhn80169
+  ⊙ #mquil07fad
   
     
   
-  ⊙ #83aqs82gho
+  ⊙ #4nfhqq566a
   
     + Adds / updates:
     
-      h
-  
-  ⊙ #eisjldgdgg
-  
-    + Adds / updates:
-    
-      g
-  
-  ⊙ #seb6uuoocl
-  
-    + Adds / updates:
-    
-      f
+      f g h
```

Comment:
Here's where we see the new behavior: `f`, `g`, and `h` are all added in one go, rather than 3 separate steps. This then impacts later namespace hashes.